packages/block-editor/src/components/default-block-appender/style.scss

```diff
@@ -112,8 +112,35 @@
 		list-style: none;
 		line-height: inherit;
 
-		.block-editor-default-block-appender__content {
-			display: block;
+		// Handle both group (block), and row/stack (flex).
+		&,
+		.block-editor-default-block-appender__content,
+		.block-editor-inserter {
+			display: inherit;
+			width: 100%;
+			flex-direction: inherit;
+			flex: 1;
+		}
+
+		// Affect Row and Stack variants.
+		// @todo: this should only affect the stack and row setup states.
+		gap: inherit;
+
+		&::after {
+			content: "";
+			display: flex;
+			border: $border-width dashed currentColor;
+			opacity: 0.4;
+			border-radius: $radius-block-ui;
+			flex: 1;
+			pointer-events: none;
+			min-height: $grid-unit-60;
+		}
+
+		// Let the parent be selectable in the placeholder area.
+		pointer-events: none;
+		.block-editor-inserter {
+			pointer-events: all;
```

Comment:
Can you clarify a bit how these styles changes work :) and if they are safe for all blocks.

Reply:
Yes indeed! So, one of the nice things about the dashed boxes this PR adds is that, at least for the Row and Stack blocks, it gives you a "safe" area to select the empty block, where right now for the base group block, the whole button takes up the space so you can't really select the block itself.

Only, this didn't work in the initial implementation, because clicking that dashed outline would select `.block-list-appender`, instead of `.block-editor-block-list__block` (which would let you select the block):
![row](https://user-images.githubusercontent.com/1204802/161058188-6c3751ce-8d70-4445-9c04-b2e10eff9267.gif)

By disabling pointer events for the placeholder stuff, then re-enabling it on the appender inside, we get the best of both worlds. I dislike touching pointer-events generally, but in this case it was the only way I could see this working.